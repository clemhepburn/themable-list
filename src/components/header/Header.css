header {
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;
  padding-right: 2rem;
  font-weight: bold;
}

header button {
  font-size: 2rem;
}