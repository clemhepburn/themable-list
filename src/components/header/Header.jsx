import React from 'react';


export default function Header() {
  return (
    <div className="header">\
      <p>Living Museum of Are.na Channels</p>

    </div>
  );
}
