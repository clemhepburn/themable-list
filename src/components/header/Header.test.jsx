import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from './Header';


describe('Header component', () => {

  afterEach(() => cleanup());

  test('renders Header', async () => {
    const { asFragment } = await render(
      <Header />);

    expect(asFragment()).toMatchSnapshot();
  });

});