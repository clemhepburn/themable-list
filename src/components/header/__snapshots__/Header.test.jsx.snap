// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders Header 1`] = `
<DocumentFragment>
  <header
    class="Header"
  >
    <p>
      Living Museum of Are.na Channels
    </p>
    
    <button></button>
    
  </header>
</DocumentFragment>
`;